.github/workflows/ci.yml

```diff
@@ -77,6 +77,12 @@ jobs:
           BUILD_S3: 1
       - name: Install test requirements
         run: pip3 install -r test/requirements.txt
+      - name: Test documentation examples
+        run: |
+          cd ./docs
+          pip3 install -r requirements.txt
+          make doctest
```

Comment:
Looks like the test is not carried out.
![image](https://user-images.githubusercontent.com/68879799/197787833-647025b5-b766-46c6-8fc7-2336f013ca76.png)

Reply:
Doctest seems to depend on the docs having been built previously. So I added a dependency on the html target into the makefile.